package.json

```diff
@@ -16,8 +16,9 @@
   },
   "dependencies": {
     "glob": "^7.1.2",
+    "rollup": "^0.60.7",
```

Comment:
This seems like it might be important to have 🤔

Reply:
Didn't need it before since I was only testing the output of my routines. But I needed to add tests that ran everything, including rollup. So now it is needed. here.